R/pkg/R/DataFrame.R

```diff
@@ -654,6 +654,33 @@ setMethod("unpersist",
             x
           })
 
+#' StorageLevel
+#'
+#' Get storage level of this SparkDataFrame.
+#'
+#' @param x the SparkDataFrame to get the storage level.
+#'
+#' @family SparkDataFrame functions
+#' @rdname storageLevel-methods
```

Comment:
this should be 
`@rdname storageLevel` instead of 
`@rdname storageLevel-methods`